src/VirtoCommerce.CatalogModule.Web/Scripts/blades/item-asset-detail.js

```diff
@@ -14,9 +14,7 @@ angular.module('virtoCommerce.catalogModule')
             $scope.languages = [];
 
             function initialize() {
-                promise.then(function(promiseData) {
-                    $scope.languages = promiseData;
-                });
+                $scope.languages = _.pluck($scope.blade.parentBlade.parentBlade.catalog.languages, 'languageCode');
```

Comment:
I think we should not assume that parentBlade.parentBlade always exists. How about fetching the catalog by Id and taking the list of languages from it?